pkg/policy/mapstate.go

```diff
@@ -1970,8 +1971,10 @@ func (ms *mapState) getIdentities(log *logrus.Logger, denied bool) (ingIdentitie
 // granularity of individual mapstate key-value pairs for both adds
 // and deletes. 'mutex' must be held for any access.
 type MapChanges struct {
-	mutex   lock.Mutex
-	changes []MapChange
+	mutex       lock.Mutex
+	changez     []MapChange
```

Comment:
changez?

Reply:
Heh :-) Renamed "temporarily" to make sure there were no unwanted referfences to it :-)

Reply:
fixed :-)